package/dist/SecureFileStorageService.js

```diff
@@ -108,8 +110,9 @@
 	        });
 	        return deferred.promise;
 	    };
-	    SecureFileStorageService.prototype.write = function (key, data) {
+	    SecureFileStorageService.prototype.write = function (key, data, iCloudSync) {
 	        var _this = this;
+	        if (iCloudSync === void 0) { iCloudSync = false; }
```

Comment:
Not a big fan of this `void 0`, people redefining undefined should be annihilated. I would prefer an `undefined` or simply a `falsy` check.